controllers/om/omclient.go

```diff
@@ -273,6 +277,21 @@ func (oc *HTTPOmConnection) GetAgentAuthMode() (string, error) {
 	return ac.Auth.AutoAuthMechanism, nil
 }
 
+func (oc *HTTPOmConnection) GetReplicaSetMemberIds() (map[string]map[string]int, error) {
```

Comment:
Do we need that separate method? Could we have an external function calling ReadDeployment and extracting the ids from the result? 

I'd prefer to limit the interface of OMConnection to the distinct API operations and have them combined in a different layer.